### ISSUE TITLE
Pay attention to accents when searching

### ISSUE THREAD

**Issue by xxxxme** [rdoeffinger/Dictionary#131]
Please pay attention to accented characters when searching.
In the Spanish <-> English dictionary, when I search for a word containing an accented character (ex: cómo), if there's another entry before it without the accent (como), I'll get that one instead. In the example I chose, the entry without the accent is 10.5 screens long on my device, which I have to scroll past to see the entry I want. With other words I'll get an entry telling me the unaccented version is a misspelling, which causes confusion.
Android 9 (LOS)

**Comment by rdoeffinger** [comment]
I thought I had fixed that (admittedly, the issue was with Russian, so might have been slightly different), need to check why it does not work.
Though for the "10 screens long" issue there is the "next word" button/menu entry at least ;)

**Comment by rdoeffinger** [comment]
As you can see I pushed a fix for it, but I am a little worried if it's really working perfectly.
Would you or anyone else be willing to test it?
I can provide a apk file either here or push a beta version to the Play store if any of those would work for you.
Can also do a debug version that can be installed in parallel to the normal one (but you might get confused which is which, since they'll have the some icon).

**Comment by rdoeffinger** [comment]
Also, it's kind of a partial fix, which could be improved but I am not sure what the best way is.
For example, if you search for cómo it will find it, but if you have written only "cóm" (which is not a word in the dictionary) it will search for the best match for "com" as before and ignore the accent. And this is a bit hard to fix as words with and without accent are mixed in the dictionary order.
Easier to fix, but not sure it's worth it/a good idea, is if you have a word with multiple accents but you get one wrong, it will fall back to a search completely ignoring accents.
In other words, this only fixes the specific case where you spelled the word fully and correctly.

**Comment by rdoeffinger** [comment]
Included in latest release.
